packages/flutter/lib/src/material/two_level_list.dart

```diff
@@ -14,19 +14,35 @@ import 'list_tile.dart';
 import 'theme.dart';
 import 'theme_data.dart';
 
+/// This enum is deprecated. Please use [ListTileTheme] instead.
+enum MaterialListType {
```

Comment:
The MaterialListType enum is not marked with @deprecated (there are doc comments to that effect though) because the analyzer complains about the enum's use in the kTileListExtent constant.